src/templates/static/legal/tac_sections/affiliate.jsx

```diff
@@ -3,121 +3,121 @@ import ListNested from '../../../_common/components/list_nested.jsx';
+                    { text: it.L('The Affiliate shall not place trades on the Affiliate\'s referrals\' behalf.') },
+                    { text: it.L('The Affiliate shall not purchase or register keywords, search terms, or other identifiers for use in any search engine, portal, sponsored advertising service, or other search or referral service that are identical or similar to any of the Company\'s trademarks or trade names that include the word "[_1]" or any variation thereof.', it.website_name) },
+                    { text: it.L('The Affiliate must ensure that the Affiliate does not place digital advertisements on websites providing unauthorised access to copyrighted content.') },
+                    { text: it.L('Neither the Affiliate, the Affiliate\'s direct relatives, nor any clients controlled by the Affiliate are eligible to become clients, and the Affiliate shall not be entitled to any share of net revenue or any other remuneration from the Company in relation to such relatives, friends, or controlled clients.') },
+                    { text: it.L('The Affiliate shall not develop and implement marketing, advertising, and promotional activities that infringe any applicable laws, rules, regulations, or codes of practice relating to marketing, advertising, and promotional activities applicable under the authority of any regulatory body of the jurisdiction(s) in which the Affiliate operates or targets business.') },
                 ]}
             >
                 <ul>
-                    <li>{it.L('In the event that the Company is aware of an Affiliate who is in breach of any of the above provisions, the Company shall have the right, in addition to any other right or remedy available to it under this Agreement or applicable law, to immediately block the Affiliate\'s access to this Programme. The Affiliate hereby irrevocably waives its rights to, and shall indemnify the Company and any member of the Binary group of companies for, any claims or demands made against the Company or any member of the Binary group of companies, their directors, officers, shareholders, employees or against the [_1] website in respect of such action taken by the Company.', it.website_name)}</li>
+                    <li>{it.L('In the event that the Company is aware of an Affiliate who is in breach of any of the above provisions, the Company shall have the right, in addition to any other right or remedy available to it under this Agreement or applicable law, to immediately block the Affiliate\'s access to this programme. The Affiliate hereby irrevocably waives its rights to, and shall indemnify, the Company and any member of the Binary group of companies for, any claims or demands made against the Company or any member of the Binary group of companies, their directors, officers, shareholders, employees or against the [_1] website in respect of such action taken by the Company.', it.website_name)}</li>
```

Comment:
@AzitaRassi "The Affiliate hereby irrevocably waives its rights... " what do you think of the use of 'its' here?

Reply:
It should be changed to "their". Nice catch.